### PR TITLE
feat(cobalt): init

### PR DESCRIPTION
**Pull request by kerichdev**
## 🎉 Theme for [cobalt.tools](https://cobalt.tools) 🎉

cobalt is your go-to place for downloads from social and media platforms. zero ads, trackers, or other creepy stuff.

image

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [submission guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md).
- [x] I have made a new directory underneath `/styles/<name-of-website>` containing the contents of the [`/template`](https://github.com/catppuccin/userstyles/blob/main/template/) directory.
  - [x] I have ensured that the new directory is in **lower-kebab-case**.
  - [x] I have followed the template and kept the preprocessor as [LESS](https://lesscss.org/#overview).
- [x] I have made sure to update the
      [`userstyles.yml`](https://github.com/catppuccin/userstyles/blob/main/scripts/userstyles.yml)
      file with information about the new userstyle.
- [x] I have included the following files:
  - `catppuccin.user.css` - all the CSS for the userstyle, based on the
    template.
  - `preview.webp` - composite image of all four individual flavor screenshots stitched together,
    generated via [Catwalk](https://github.com/catppuccin/toolbox/tree/main/catwalk#readme).

### CODE REVIEW

**Review by uncenter — CHANGES_REQUESTED** [review]
Name should just be `cobalt`, no need to include the domain. This requires changing the directory name as well as the userstyles.yml and the userstyle CSS metadata.

**Review by uncenter — CHANGES_REQUESTED** [review]
Screenshot 2024-06-05 at 10 19 34 (Arc)

Popup backdrop/box-shadow isn't clear enough to distinguish the modal from the background.

**Comment by kerichdev** [comment]
oops, missed the name issue, though
thought it goes along the same way with the .io websites

**Comment by uncenter** [comment]
> oops, missed the name issue, though thought it goes along the same way with the .io websites
> 
> cobalt seems like too common of a name to simply use it without the domain name

We don't have any other userstyles with the name `cobalt` and the website itself refers to it as `cobalt`, not `cobalt.tools`.

**Review by isabelroses — APPROVED** [review]

File: scripts/userstyles.yml [hunk 1]
```diff
@@ -237,6 +240,13 @@ userstyles:
       app-link: "https://codeberg.org"
     current-maintainers: [*isabelroses]
     past-maintainers: [*justtobbi]
+  cobalt:
+    name: cobalt
+    categories: [productivity]
```
Comment (uncenter) [inline scripts/userstyles.yml:245]:
```suggestion
    categories: [productivity, photo_and_video]
```

File: styles/cobalt/catppuccin.user.css [hunk 2]
```diff
@@ -0,0 +1,93 @@
+/* ==UserStyle==
+@name cobalt Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/cobalt
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/cobalt
+@version 1.0.0
```
Comment (uncenter) [inline styles/cobalt/catppuccin.user.css:5]:
```suggestion
@version 0.1.0
```

File: styles/cobalt/catppuccin.user.css [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document domain('cobalt.tools') {
+
+  :root[data-theme="dark"] {
```
Comment (uncenter) [inline styles/cobalt/catppuccin.user.css:20]:
```suggestion
@-moz-document domain('cobalt.tools') {
  :root[data-theme="dark"] {
```

Sorry for the whitespace nit :)

**Review by uncenter — CHANGES_REQUESTED** [review]

**Review by uncenter — APPROVED** [review]

**Pull request merged**